code/game/objects/items/grenades/chem_grenade.dm

```diff
@@ -623,3 +623,22 @@
 
 	beakers += B1
 	beakers += B2
+
+
+/obj/item/grenade/chem_grenade/blood
```

Comment:
undocumented addition, remove if this was just for testing

Reply:
admins can play with it and i'd like to keep it for testing

Reply:
> admins can play with it and i'd like to keep it for testing

add it to the name/desc/changelog of the PR then :^)

Reply:
no problem, will add to changelog